source/unified-test-format/tests/invalid/initialCollectionData-additionalProperties.yml

```diff
@@ -1,6 +1,6 @@
-description: "collectionData-additionalProperties"
+description: "initialCollectionData-additionalProperties"
 
-schemaVersion: "1.0"
+schemaVersion: "1.5"
```

Comment:
I don't think anything here pertains to newer schema versions, since renaming `collectionData` to `initialCollectionData` is strictly internal refactoring. I propose leaving this as 1.0.